jrnl/plugins/fancy_exporter.py

```diff
@@ -70,6 +75,13 @@ def export_entry(cls, entry):
         card.append(cls.border_l + cls.border_b * body_linewrap + cls.border_m)
         return "\n".join(card)
 
+    @classmethod
+    def check_linewrap(cls, linewrap, card):
+        if len(card[0]) > linewrap:
+            raise JrnlError("LineWrapTooSmallForDateFormat", config_linewrap=linewrap)
+        else:
+            pass
+
```

Comment:
I don't think this method needs to exist. Normally I like breaking things down into individual methods as much as possible, but it seems easier to read the conditional and the error raise (without needing the else: pass at all) in the calling method. Or, if the method does need to exist, maybe specify what is being checked.

Reply:
hmmm... my only reason to break it up this way was to unittest it. That said, I agree that it doesn't need to be a *method*. I think it would make sense as a free function. 

I would prefer it stay as its own piece of logic for testability/reuse/maintainability.

I have updated the signature to better express intent.

Reply:
Those are all great points! Thanks for these changes.